.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: main
+
+on: push
+
+jobs:
+  test:
+    # Skip this job if the head commit’s message contains the string “[skip ci]”.
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        # https://github.com/actions/checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        # https://github.com/actions/setup-node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
```

Comment:
Why are we using different version in `nvmrc` https://github.com/kumahq/kuma-gui/blob/master/.nvmrc#L1 ? shouldn't this be the same?

Reply:
I intend to lift the versions of Node in general, but I want to do this separately for Netlify deployment previews (which use the `.nvmrc` file).